pkg/test/framework/test.go

```diff
@@ -208,6 +208,14 @@ func (t *Test) runInternal(fn func(ctx TestContext), parallel bool) {
 		return
 	}
 
+	// TODO: should we also block new cases?
+	suiteName := t.s.settings.TestID
+	if len(t.featureLabels) < 1 && !features.GlobalWhitelist.Contains(suiteName, t.name) {
+		scopes.Framework.Errorf("Detected new test %s in suite %s with no feature labels.  "+
+			"See istio/istio/pkg/test/framework/features/README.md", suiteName, t.name)
+		t.goTest.FailNow()
```

Comment:
can you do goTest.Fatal("error") the it will be output more clearly I think?

Reply:
Thank you.  I knew there was a function like that, but couldn't find it..